doc/user-guide/Getting-started.md

```diff
@@ -10,7 +10,7 @@ This short guide will show you how to compile MongooseIM from source code or ins
 
 MongooseIM is supported on Mac OS X 10.6.8 and later. Before you can compile and run MongooseIM, you also need the following to be installed on your system:
 
-* GNU Make and GCC (the GNU Compiler Collection). To ensure that these are installed, you can install the Command Line Tools for Xcode, available via Xcode or from the Apple Developer website.
```

Comment:
I think this should remain "install". By default the Command Line Tools for Xcode are not installed, in order to use them one needs to install them.